### PR TITLE
Add html file

### PR DESCRIPTION
**Pull request by principles31**
**Awesome books: refactor to use JavaScript classes**

- The goal of this project is to refactor the Awesome books application to use JavaScript classes.
- **Project requirements**
. Use JavaScript classes instead of objects and arrays.
. You can use objects and arrays, but do it inside classes.
. Create class methods to add and remove books.
. Maintain all of the original functionality of the application (i.e. no bugs were introduced while refactoring).
- A screenshot of the has been included
- 
Screenshot from 2022-05-04 13-26-10

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -62,4 +62,70 @@ bookList.addEventListener('click', (e) => {
     showListBooks();
   }
 });
-document.addEventListener('DOMContentLoaded', showListBooks);
\ No newline at end of file
+document.addEventListener('DOMContentLoaded', showListBooks);
+
```
Comment (TedLivist) [inline index.js]: - [x] Let's keep our code neat. You have a lot of unnecessary code in this file. Kindly delete from line 16 to this line.

File: index.js [hunk 2]
```diff
@@ -62,4 +62,70 @@ bookList.addEventListener('click', (e) => {
...
+           const filteredBooks = this.listBooks.filter((book) => book.id !== +id);
+           this.listbooks = JSON.parse(localStorage.getItem('books'));
+           localStorage.setItem(
+             'books',
+             JSON.stringify(filteredBooks),
+           );
+           this.showListBooks();
+         }
+       }
+}
```
Comment (TedLivist) [inline index.js]: - [x] **[OPTIONAL]** I would recommend you indent your code properly.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @principles31 and @Smart1-hub,

Good job so far!
✔️ Good documentation
✔️ Refactor to use classes
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by principles31 — COMMENTED** [review]
- unnecessary code has been removed.
-  Proper indentation has been implemented as requested by the previous reviewer.

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @principles31, and @Smart1-hub👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**